### PR TITLE
Update package versions: RADICAL-Cybertools (RE, RG, RP, RS, RU)

### PR DESCRIPTION
**Pull request by mtitov**
RADICAL-EnTK, RADICAL-GTOD, RADICAL-Pilot, RADICAL-SAGA, RADICAL-Utils

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-radical-entk/package.py [hunk 1]
```diff
@@ -31,6 +32,9 @@ class PyRadicalEntk(PythonPackage):
     depends_on("py-radical-utils@1.12:", type=("build", "run"), when="@1.12:")
     depends_on("py-radical-pilot@1.12:", type=("build", "run"), when="@1.12:")
 
+    depends_on("py-radical-utils@:1.11", type=("build", "run"), when="@:1.11")
+    depends_on("py-radical-pilot@:1.11", type=("build", "run"), when="@:1.11")
```
Comment (andre-merzky) [inline var/spack/repos/builtin/packages/py-radical-entk/package.py:36]: This looks like a duplicated line?  Otherwise LGTM - thanks!

**Review by andre-merzky — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-radical-entk/package.py (hunk 1 above)
Comment (mtitov) [reply var/spack/repos/builtin/packages/py-radical-entk/package.py:36]: That just to enforce particular versions (first pair for versions >=1.12, and the next pair for versions <=1.11)

**Review by tldahlgren — APPROVED** [review]
Confirmed all new sha256.  Dependency ranges LGTM.

File: var/spack/repos/builtin/packages/py-radical-entk/package.py (hunk 1 above)
Comment (andre-merzky) [reply var/spack/repos/builtin/packages/py-radical-entk/package.py:36]: Ah, sorry, the lines are in fact different (`utils` vs. `pilot`).  Sorry for the noise...

**Review by andre-merzky — APPROVED** [review]

**Pull request merged**